servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -575,6 +576,7 @@ public MergeResponse transplantCommitsIntoBranch(
     try {
       checkArgument(!hashesToTransplant.isEmpty(), "No hashes given to transplant.");
       validateCommitMeta(commitMeta);
+      validateNoRelativeSpec(expectedHash);
```

Comment:
When we get here, hashes were already validated in `ImmutableMerge` and `ImmutableTransplant` objects (see their check methods) – but I figured it doesn't hurt to re-validate.